projects/angular/src/emphasis/alert/alert.ts

```diff
@@ -15,9 +25,9 @@ import { MultiAlertService } from './providers/multi-alert.service';
   selector: 'clr-alert',
   providers: [AlertIconAndTypesService],
   templateUrl: './alert.html',
-  styles: [':host { display: block; }'],
 })
 export class ClrAlert implements OnInit, OnDestroy {
+  @Input('clrAlertLightweight') isLight = false;
```

Comment:
In theory we should set `clrAlertClosable=false` for all `clrAlertLightweight=true`, but I prefer not to extend the scope of this PR that much. We might need another JIRA ticket or discussion regarding such a change.

Reply:
Used display: none for hiding it for now, we can add a separate logic for block all the closing for lightweight alerts.